tsconfig.json

```diff
@@ -24,5 +24,15 @@
     "./typings/*",
     "./.storybook/docs/components/*.tsx"
   ],
+  "watchOptions": {
```

Comment:
This are some of the "boilerplate" commands, but they seemed to make sense to use them.